applications/benchmark/CMakeLists.txt

```diff
@@ -38,4 +38,11 @@ target_include_directories(NewtonRaphsonBenchmark
     PUBLIC ${CMAKE_SOURCE_DIR}/src)
 append_to_benchmarks(NewtonRaphsonBenchmark)
 
+add_executable(BlockVectors
+    EXCLUDE_FROM_ALL BlockVectors.cpp)
+target_include_directories(BlockVectors
+    PUBLIC ${CMAKE_SOURCE_DIR}/src)
```

Comment:
Can be made `PRIVATE`. Not that it matters much, but `PUBLIC` means "this target needs these includes to compile, and targets that link to it do as well", and since you're probably not linking this executable to anything else, `PRIVATE` ("this target needs these includes to compile") is enough. Not terribly important, just FYI.

Reply:
Good point, thanks. I just copied it from somewhere without understanding it.